src/app/services/session-storage.service.ts

```diff
@@ -20,15 +20,30 @@ export class SessionStorageService {
     else this._data = JSON.parse(dataRaw);
   }
 
+  /**
+   * Updates sessions storage
+   */
   updateSessionData(key: string, value: any): void {
     if (value != null) this._data[key] = value;
     else delete this._data[key];
 
     sessionStorage.setItem('fem-alert', JSON.stringify(this.serialize(this._data)));
   }
 
+  /**
+   * Clears session storage
+   */
+  clearSessionData(): void {
+    sessionStorage.clear();
+  }
+
   getSessionData(key: string): any{
     let storageData = JSON.parse(sessionStorage.getItem('fem-alert'));
```

Comment:
Here you could simply return a attribute of the `_data` property of the service

Reply:
that is true didn't think of it because im used to getting everything straight from storage